app/service/utils.go

```diff
@@ -55,6 +57,16 @@ func ResolveURLQueryGetBoolField(httpReq *http.Request, name string) (bool, erro
 	return strValue == "1", nil
 }
 
+// ResolveURLQueryPathInt64Field extracts a path element of type int64 from the query
+func ResolveURLQueryPathInt64Field(httpReq *http.Request, name string) (int64, error) {
```

Comment:
It is time now to start adding unit tests each time you add a new function 😋 !

Reply:
Sure. Why not? I love them! 👍

Reply:
done